website/docs/addon_applications_admin.md

```diff
@@ -1,26 +1,230 @@
+   2. **Selected:** when chosen, the Applications list will show up.
+3. **Applications:** List of applications available for profile filter.
+4. **+ button:** Add new profile filter.
+
+:::tip Default studio profile
+![](assets/applications/default_application_filters.png)
+Using default settings will show all applications for all task types.
+It is recommended to have one default profile without task types filter, with standard set of applications used in the studio.
+:::
+
+### Tools Definitions
+
+A tool in AYON is anything that needs to be selectively added to your DCC applications. Most often these are plugins, modules, extensions or similar depending on what your package happens to call it.
+
+![](assets/applications/tools_defimitions.png)
+> Tools Definitions settings are much similar to Applications Definitions settings.
+
+1. **Tool Group:**
+   1. **Name:** Tool name.
+   2. **Label:** Tool label. Name is used if not filled.
+   3. **Environments:** Environments variables that will be applied to all tool variants.
+2. **Tool Variants:**
+   1. **Name:** Tool variant name.
+   2. **Label:** Tool variant label. Name is used if not filled.
+   3. **Hosts:** List of host names you want the tool variant to work with.
+   4. **Applications:** Filter applications for which is tool applicable.
+   5. **Environments:** tool variant environments
+   6. **+ button:** Add new tool variant.
+3. **+ button:** Add new tool group.
+
+:::tip Tool Examples
+
+AYON comes with some major CG renderers pre-configured as an example, but these and any others will need to be changed to match your particular environment.
+
+![](assets/applications/examples_tools.png)
+
+<details><summary>Pre-configured Example - Autodesk Arnold</summary>
+
+![tools](assets/applications/arnold_tool_example.png)
+
+In this example you can see that we set most of the environment variables in the general MTOA level, and only specify the version variable in the individual versions below. Because all environments within AYON setting will resolve any cross references, this is enough to get a fully dynamic plugin loading as far as your folder structure where you store the plugins is nicely organized.
+
+MTOA will automatically set the `MAYA_VERSION`(which is set by Maya Application environment) and `MTOA_VERSION` into the `MTOA` variable. We then use the `MTOA` to set all the other variables needed for it to function within Maya.
+</details>
+:::
+
+### Tools filters
+Tools that will be used on application launch. It is profile based filtering that allows to filter tools for a context. Tools can have more specific filtering than applications.
+
+All the tools defined in Tools Definitions can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
```

Comment:
```suggestion
All the tools defined in Tools Definitions can then be assigned to projects, certain task types or even selectively for only certain folders. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
```
Somehow the first sentence made it seem like you assign it per project - and then the second made it seem like you'd use the filtering per e.g. folder or task only for "version differences" but that's wrong, because you _may_ e.g. also enable a plug-in only for animation task type across the full project, of just one version.

Reply:
https://github.com/ynput/ayon-documentation/pull/220/commits/6487b573c9ba30203ae4fbf2a4c6b517764b9e02